wolfcrypt/src/asn.c

```diff
@@ -8076,10 +8097,7 @@ int ParseCertRelative(DecodedCert* cert, int type, int verify, void* cm)
 
         #ifdef HAVE_OCSP
             /* Need the CA's public key hash for OCSP */
-            ret = CalcHashId(cert->ca->publicKey, cert->ca->pubKeySize,
-                                                           cert->issuerKeyHash);
-            if (ret != 0)
-                return ret;
+            XMEMCPY(cert->issuerKeyHash, cert->ca->subjectKeyHash, KEYID_SIZE);
```

Comment:
issuerKeyHash is the hash of the Issuer's public key. The hash **shall** be **calculated** over the value (excluding tag and length) of the subject public key field in the issuer's certificate. - https://tools.ietf.org/html/rfc2560#section-4

Reply:
When we decode a certificate and call `GetKey()` we are hashing the public key and caching that hash in the decoded certificate record. That cached hash is copied over to the Signer record when the decoded cert is added to the cert manager.